docs/sql-guide/functions/function-datetimefromparts.md

```diff
@@ -0,0 +1,55 @@
+---
+title: DATETIMEFROMPARTS()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# DATETIMEFROMPARTS() function
+
+The `DATETIMEFROMPARTS()` function takes integer date and time arguments and returns timestamp.
+
+## Syntax
+
+```
+DATETIMEFROMPARTS(int_year, int_month, int_date, int_hour, int_minute, int_seconds, int_milliseconds)
+```
+
+## Arguments
+
+| Argument         | Data type | Description                                                                                     | Required | Further information |
+|------------------|-----------|-------------------------------------------------------------------------------------------------|---|---|
+| int_year         | integer   | Integer value that specifies year [0,9999]                                                      | Yes | |
```

Comment:
The same goes here. 
* Don't add extra spaces because they can conceal hidden characters that break the table rendering.
* A single space either side of whatever you're writing is fine.